CHANGES

```diff
@@ -46,6 +46,10 @@ Features added
 * #9391: texinfo: improve variable in ``samp`` role
 * #9578: texinfo: Add :confval:`texinfo_cross_references` to disable cross
   references for readability with standalone readers
+* #9337: Add :confval:`search_with_keys` that enables :kbd:'/' as
```

Comment:
```suggestion
* #9337: HTML theme, add option ``search_with_keys`` that enables :kbd:'/' as
```

Reply:
They are technically not "configuration variables" as that would be those in ``conf.py``.